core/preloop.gms

```diff
@@ -188,4 +188,9 @@ if (cm_startyear gt 2005,
 execute_load "input_ref.gdx", pm_PEPrice, pm_SEPrice, pm_FEPrice;
 );
 
+*** load vm_capEarlyReti(ttot,regi,te) from reference gdx to have a reference point for q_smoothphaseoutCapEarlyReti and q_limitCapEarlyReti
```

Comment:
maybe we should think about adding the feature of suing an alternative reference gdx to load these values to avoid the issue we had with the outdated feasibility frontier runs?

Reply:
not sure I understand?

Reply:
sorry, my comment was related with the `q_eqadj` and has no connection with this PR. We can talk about it a bit further the next time we meet.